agent/index.html

```diff
@@ -0,0 +1,25 @@
+<!doctype html>
+
+<html>
+  <head>
+    <title>BigTest</title>
+    <style>
+      body, html {
+        margin: 0;
+        padding: 0;
+        height: 100%;
+      }
+
+      #test-frame {
+        border: none;
+        height: 100%;
+        width: 100%;
+      }
+    </style>
+  </head>
+
+  <body>
+    <iframe id="test-frame"></iframe>
+    <script src="./agent.ts"></script>
+  </body>
+</html>
```

Comment:
One thing to note here is that I put this in `./agent` rather than `./src/agent` which was my first choice. The reason is that the watch command was triggering rebuilds when changing the agent code, but this is unnecessary and slow, since we already have watch and rebuild via parcel. Rather than somehow exclude this directory it felt easier to just move it outside `src`.

Reply:
I think this is actually ok. We may want to actually ship it as a separate npm package some day.